code/js/popup.js

```diff
@@ -154,9 +154,9 @@ function dumpBookmarks(query)
 					    {	  	
 	
 						   	if(!favs.includes(fname)) {
-						   		inside_text = inside_text + "<a  target='_blank' href='/code/"+key+"/"+obj[key][dd]+"'>"+sub_result_number+". "+obj[key][dd]+"</a>"+"&nbsp;&nbsp;<i id='myStar"+temp2+"\' class='fa fa-star'></i><br>";
+						   		inside_text = inside_text + "<a  target='_blank' href='/code/"+key+"/"+obj[key][dd]+"'>"+sub_result_number+". "+obj[key][dd]+"</a>"+"&nbsp;&nbsp;<a style='color:inherit' href='/code/"+key+"/"+obj[key][dd]+"'download><i id='myDownload"+temp2+"\' style='float:right' class='fa fa-download'></i></a><i id='myStar"+temp+"\' style='float:right;width:8%' class='fa fa-star'></i><br>";
```

Comment:
It looks like `temp` should be replaced by `temp2`. This is breaking the favorites features. Same is applicable to the else section. :+1: 

Kindly take a look into this. :+1:

Reply:
Oops yes! Made the corresponding changes.